site/ko/swift/python_interoperability.ipynb

```diff
@@ -0,0 +1,687 @@
+        }
+      },
+      "source": [
+        "import Python\n",
+        "print(Python.version)"
+      ],
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "3.6.8 (default, Jan 14 2019, 11:02:34) \r\n",
+            "[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]]\r\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SCrJ-r9Vum5-",
+        "colab_type": "text"
+      },
+      "source": [
+        "## 파이썬 버전 설정"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "hCvI1pyhum5_",
+        "colab_type": "text"
+      },
+      "source": [
+        "기본적으로 파이썬을 임포트 하면, 스위프트는 시스템 라이브러리 경로에 따라 설치된 최신 버전의 파이썬을 검색합니다.\n",
+        "특정한 파이썬을 설치하고자 할 때, 설치에서 제공하는 `libpython` 공유 라이브러리에 `PYTHON_LIBRARY` 환경변수를 설정해야 합니다. 예를 들어:\n",
+        "\n",
+        "`export PYTHON_LIBRARY=\"~/anaconda3/lib/libpython3.7m.so\"`\n",
+        "\n",
+        "정확한 파일명은 파이썬 환경과 플랫폼마다 다를 수 있습니다."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PKylwmZtum6A",
+        "colab_type": "text"
+      },
+      "source": [
+        "또는 스위프트에 일치하는 파이썬 버전에 대한 시스템 라이브러리 경로를 검색하도록 알려주는 `PYTHON_VERSION` 환경변수를 설정할 수 있습니다. `PYTHON_LIBRARY`가 `PYTHON_VERSION`보다 우선한다는 점을 유의해야 합니다.\n",
+        "\n",
+        "또한 코드에서 `PYTHON_VERSION` 설정과 동일한 `PythonLibrary.useVersion`함수를 호출할 수 있습니다."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "XyFJzOlAum6B",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "// PythonLibrary.useVersion(2)\n",
+        "// PythonLibrary.useVersion(3, 7)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bllCt2ltum6D",
+        "colab_type": "text"
+      },
+      "source": [
+        "_Note: \n",
+        "파이썬을 임포트한 직후, 파이썬 코드를 호출하기 전에 `PythonLibrary.useVersion`을 실행해야 합니다. 이것은 파이썬 버전을 동적으로 바꾸는 데 사용될 수 없습니다._\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PiJyhOmUum6D",
+        "colab_type": "text"
+      },
+      "source": [
+        "[파이썬 라이브러리 로딩 과정에서 생성되는 디버그 출력](https://github.com/apple/swift/pull/20674#discussion_r235207008)을 확인하기 위해서 `PYTHON_LOADER_LOGGING=1` 설정하세요."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "rU0WY_sJodio"
+      },
+      "source": [
+        "## 기본\n",
+        "스위프트에서 `PythonObject`는 파이썬 객체를 나타냅니다.\n",
+        "모든 파이썬 API는 `PythonObject` 인스턴스를 사용하거나 반환합니다.\n",
+        "\n",
+        "스위프트에서 기본형(숫자 및 배열처럼)은 `PythonObject`로 전환할 수 있습니다. 몇몇 경우에 (\"PythonConvertible\" 인수를 받는 리터럴과 함수의 경우), 변환이 암묵적으로 일어납니다. 스위프트 값을 `PythonObject`에 명시적으로 지정하려면 `PythonObject` 이니셜라이저를 사용합니다.\n",
```

Comment:
\"PythonConvertible\"  => \`PythonConvertible\`

Reply:
Thanks for reviewing.
I modified it.